lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -22,8 +22,11 @@ import { FilterParamsModel, FilterRepresentationModel } from '../models/filter.m
 import { TaskFilterService } from './../services/task-filter.service';
 import { TaskListService } from './../services/tasklist.service';
 
+/**
+ * @deprecated: 'adf-filters' and 'taskListService-filters' selectors are deprecated, use adf-task-filters instead.
```

Comment:
can you add in which version we deprecate it? Please see the other deprecated comments